projects/github-actions/test-results-to-slack/README.md

```diff
@@ -55,9 +58,68 @@ The action relies on the following parameters.
 
 - (Required) `github_token` is a GitHub Access Token used to access GitHub's API. The token should be stored in a [secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository)
 - (Required) `slack_token` is the Auth token of a bot that is installed on your Slack workspace. The token should be stored in a [secret](https://docs.github.com/en/actions/reference/encrypted-secrets#creating-encrypted-secrets-for-a-repository). See the [instructions here](https://slack.com/intl/en-hu/help/articles/115005265703-Create-a-bot-for-your-workspace) on how to create a bot.
-- (Required) `slack_channel` is the Slack channel ID where the messages will be sent to.
+- (Required) `slack_channel` is the Slack channel ID where the messages will be sent to. See below how to find the channel ID. Check the channel's details in you Slack app to get find the channel ID.
```

Comment:
```suggestion
- (Required) `slack_channel` is the Slack channel ID where the messages will be sent to. See below how to find the channel ID. Check the channel's details in your Slack app to find the channel ID.
```

Reply:
Done and done with [0ab3521](https://github.com/Automattic/jetpack/pull/25830/commits/0ab352190a6f86b2ddf795f37064f0ebecb5ce70)

Reply:
And with [1f5660a](https://github.com/Automattic/jetpack/pull/25830/commits/1f5660a69278bec6df1ae7f642e12b1c1efae989). Apparently I need 2 commits to fix a typo.